### PR TITLE
Update react/react-dom to latest stable minor release

### LINKED ISSUE #475: Update react / react-dom peer dependency to latest stable release
# Feature Request

## Description
Update [react / react-dom peer dependency to latest stable release](https://github.com/facebook/react/releases)

### PR DESCRIPTION
**Pull request by bjankord**
### Summary
Resolves #475

### CODE REVIEW

File: packages/terra-clinical-detail-view/package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
   },
   "homepage": "https://github.com/cerner/terra-clinical#readme",
   "peerDependencies": {
-    "react": "^16.4.2",
+    "react": "^16.8.5",
     "react-dom": "^16.4.2"
```
Comment (tbiethman) [inline packages/terra-clinical-detail-view/package.json]: Wanna get this feller?

File: packages/terra-clinical-header/package.json [hunk 2]
```diff
@@ -22,7 +22,7 @@
   },
   "homepage": "https://github.com/cerner/terra-clinical#readme",
   "peerDependencies": {
-    "react": "^16.4.2",
+    "react": "^16.8.5",
     "react-dom": "^16.4.2"
```
Comment (tbiethman) [inline packages/terra-clinical-header/package.json]: This one (and item collection, item display, item view, and label value view) too

File: packages/terra-clinical-detail-view/package.json (hunk 1 above)
Comment (bjankord) [reply packages/terra-clinical-detail-view/package.json]: https://github.com/cerner/terra-clinical/pull/476/commits/ca2253ebee49b5a308b5906f148ffb7a1af57ee7

File: packages/terra-clinical-header/package.json (hunk 2 above)
Comment (bjankord) [reply packages/terra-clinical-header/package.json]: https://github.com/cerner/terra-clinical/pull/476/commits/ca2253ebee49b5a308b5906f148ffb7a1af57ee7

**Pull request merged**